scripts/leetcode.js

```diff
@@ -608,25 +611,34 @@ function startUploadCountDown() {
 }
 
 /* we will need specific anchor element that is specific to the page you are in Eg. Explore */
-function insertToAnchorElement(elem){
-  if(document.URL.startsWith("https://leetcode.com/explore/")) {
+function insertToAnchorElement(elem) {
+  if (document.URL.startsWith("https://leetcode.com/explore/")) {
     // means we are in explore page
-    target = document.getElementsByClassName('action')[0].getElementsByClassName('row')[0].getElementsByClassName('col-sm-6')[1]
-    elem.className = "pull-left"
-    target.childNodes[0].prepend(elem)
-
+    action = document.getElementsByClassName('action');
+    if (checkElem(action)
+      && checkElem(action[0].getElementsByClassName('row'))
+      && checkElem(action[0].getElementsByClassName('row')[0].getElementsByClassName('col-sm-6'))
+      && action[0].getElementsByClassName('row')[0].getElementsByClassName('col-sm-6').length > 1) {
+      target = action[0].getElementsByClassName('row')[0].getElementsByClassName('col-sm-6')[1]
+      elem.className = "pull-left"
+      if (target.childNodes.length > 0)
+        target.childNodes[0].prepend(elem)
+    }
   } else {
-    target = document.getElementsByClassName('action__38Xc')[0] 
-    elem.className = "runcode-wrapper__8rXm"
-    target.childNodes[0].prepend(elem)
+    if (checkElem(document.getElementsByClassName('action__38Xc'))) {
+      target = document.getElementsByClassName('action__38Xc')[0]
+      elem.className = "runcode-wrapper__8rXm"
+      if (target.childNodes.length > 0)
+        target.childNodes[0].prepend(elem)
+    }
   }
 }
 
 /* start upload will inject a spinner on left side to the "Run Code" button */
 function startUpload() {
   try {
     elem = document.getElementById('leethub_progress_anchor_element')
-    if (elem !== undefined) {
+    if (!elem) {
```

Comment:
this seems the opposite to what was done before. `!elem` tells me that if that element is not present, create the spinner. But previously, you had it as if the element was present, create a spinner.
reason? @sbibek